data/human/human missions.txt

```diff
@@ -7592,7 +7593,8 @@ mission "Timothy Radrickson 2: Good Interlude"
 	to offer
 		has "event: timothy radrickson: on rand"
 	on offer
-		log `Despite starting over with nothing, Timothy the Stowaway has sent me credits to pay me back for that passage I gave him over a year ago.`
+		log "People" "Timothy Radickson" `A stowaway I found on my ship. He requested I take him to Rand to start a new life, and I did. He promised to pay me back. And did so, sending me 50,000 credits.`
```

Comment:
Just so you know, it appends this to what's already in the log entry. So you'll end up with duplication of information.

Reply:
so... should it not be removed? Or.. what?